channel.go

```diff
@@ -441,6 +445,17 @@ func (ch *Channel) Connect(ctx context.Context, hostPort string, connectionOptio
 	return c, err
 }
 
+// exchangenUpdated updates the peer heap.
+func (ch *Channel) exchangenUpdated(c *Connection) {
+	if c.remotePeerInfo.HostPort == "" {
+		// Hostport is unknown until we get init resp.
+		return
+	}
+
+	p := ch.rootPeers().GetOrAdd(c.remotePeerInfo.HostPort)
+	ch.subChannels.updatePeerHeap(p)
```

Comment:
since `updatePeerHeap` only cares about the host:port, should we just pass host:port directly?